com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/CheckForOperationsInProgressStepTest.java

```diff
@@ -0,0 +1,119 @@
+package com.sap.cloud.lm.sl.cf.process.steps;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Stream;
+
+import org.cloudfoundry.client.lib.domain.CloudService;
+import org.cloudfoundry.client.lib.domain.ImmutableCloudService;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mock;
+
+import com.sap.cloud.lm.sl.cf.client.lib.domain.CloudServiceExtended;
+import com.sap.cloud.lm.sl.cf.core.cf.clients.EventsGetter;
+import com.sap.cloud.lm.sl.cf.core.cf.clients.ServiceGetter;
+import com.sap.cloud.lm.sl.cf.core.cf.services.ServiceOperationType;
+import com.sap.cloud.lm.sl.common.util.JsonUtil;
+import com.sap.cloud.lm.sl.common.util.TestUtil;
+
+public class CheckForOperationsInProgressStepTest extends SyncFlowableStepTest<CheckForOperationsInProgressStep> {
+
+    protected static final String TEST_SPACE_ID = "test";
+
+    @Mock
+    private ServiceGetter serviceInstanceGetter;
+    @Mock
+    private EventsGetter eventsGetter;
+
+    protected StepInput stepInput;
+
+    public static Stream<Arguments> testExecute() {
```

Comment:
Is there any way to avoid using jsons here? You have small components which could be easily created here?

Reply:
fixed